sdk/eventgrid/azure-eventgrid/samples/async_samples/sample_authentication_async.py

```diff
@@ -20,8 +20,8 @@
 from azure.eventgrid.aio import EventGridPublisherClient
 from azure.core.credentials import AzureKeyCredential
 
-topic_key = os.environ["EVENTGRID_TOPIC_KEY"]
-endpoint = os.environ["EVENTGRID_TOPIC_ENDPOINT"]
+topic_key = os.environ["EVENTGRID_TOPIC_KEY"] or "your-eventgrid-topic-key"
+endpoint = os.environ["EVENTGRID_TOPIC_ENDPOINT"] or "your-eventgrid-topic-endpoint"
```

Comment:
We generally try to keep a PR focused to one library at a time. Can we revert these changes for eventgrid?

Reply:
Done